rule/add_constant.go

```diff
@@ -36,13 +38,16 @@ type AddConstantRule struct {
 	allowList       allowList
 	ignoreFunctions []*regexp.Regexp
 	strLitLimit     int
-
-	configureOnce sync.Once
+	configureOnce   sync.Once
```

Comment:
Please do not remove an empty line. It's added by purpose to separate rule's arguments with internal fields.